### DISCUSSION TITLE
Spatiotemporal Scales as a way of discriminating analyses, tools and datasets.

### DISCUSSION THREAD

**Discussion by joewheaton** [Riverscapes/riverscapes-tools#676] [General]
As we work on concepts for discriminating tools, Spatial and Temporal scales has been one we deal with a lot. 

![Riverscapes Consortium - Scale of Analysis   Tools](https://user-images.githubusercontent.com/9723905/207051539-1336004a-f6ee-438c-82c4-f91454bbc10f.png)

I will stub out a page on the riverscapes.net site for this, but I wanted to see if any of the @Riverscapes/riverscape-analysts, @Riverscapes/riverscape-project-curators or @Riverscapes/developers had any concerns with any of this (or suggestions).

**Comment by lauren-herbine** [comment]
* Should "Cell" in the Resolution column also say "Cell (Raster)"?
* Catchment and Sub-Catchment are defined by the same range (10^1 to 10^3 km2)- is that on purpose?
* Starting with Cell/GU in resolution, and then also ending with Cell is a little confusing...
* "Snapshot" is supposed to be the shortest on the temporal scale? As it is capturing something at one instant, instead of things along the arrow which are observed over time?
